src/components/Footer.js

```diff
@@ -0,0 +1,37 @@
+import "./Footer.css";
+
+function Footer() {
+    return (
+        <div className="footer">
```

Comment:
시맨틱하게 표현해주면 좋을 것 같네요~
```suggestion
        <footer className="footer">
```